.storybook/Card.js

```diff
@@ -3,6 +3,54 @@ import { storiesOf } from '@storybook/react'
 import { Box, Card } from '../src'
 
 storiesOf('Card', module)
+  .add('Box Shadows without border', () => (
+    <Box>
+      <Card
+        boxShadowSize="sm"
+        m={4}
+        p={4}
+        width={1 / 2}
+        color="text"
+        bg="white"
+        borderWidth={0}
```

Comment:
Do we intend to encourage using the Card without the border? Not sure I understand the intent here...